js/projects_info/project.js

```diff
@@ -0,0 +1,7 @@
+function project(name, imageLocation, link, description, languages) {
```

Comment:
In JS, once you get over a few arguments (2 or 3), it's typically better to use an object to be passed in, so it's easier to update and maintain in the future. Since the amount of arguments, and their order could change in the function, and we would have to carefully update all code in the project referencing it if it was passing in lots of arguments.

```js
function project (params) {
    this.name = params.name;
    this.imageLocation = params.imageLocation;
    this.link = params.link;
    this.description = params.description;
    this.languages = params.languages;
}

var blogList = {
    'gavynsBlog': {
        'name': 'Gav\'s Dev Blog',
        'imageLocation': './images/blog_images/gavs_dev_blog.jpg',
        'link': 'https://gavsdevblog.wordpress.com',
        'description': 'New Description',
        'languages': ['HTML', 'CSS', 'JS']
    },
    'gwensBlog': {
        'name': 'Gwen Faraday: Coding Blog',
        'imageLocation': './images/blog_images/gwen_blog.jpg',
        'link': 'https://fake.wordpress.com',
        'description': 'New Description',
        'languages': ['HTML', 'CSS', 'JS']
    }
};

projects.push(new project(blogList.gavynsBlog));
projects.push(new project(blogList.gwensBlog));
```